neural_lam/utils.py

```diff
@@ -271,3 +275,49 @@ def init_wandb_metrics(wandb_logger):
     experiment.define_metric("val_mean_loss", summary="min")
     for step in constants.VAL_STEP_LOG_ERRORS:
         experiment.define_metric(f"val_loss_unroll{step}", summary="min")
+
+
+@rank_zero_only
+def rank_zero_print(*args, **kwargs):
+    """Print only from rank 0 process"""
+    print(*args, **kwargs)
+
+
+@rank_zero_only
```

Comment:
This is super nice, I was not aware that this util exists! Probably other places in the code where this could be used as well, but that does not have to be in this PR.